.github/workflows/deploy-eks.yaml

```diff
@@ -85,3 +86,11 @@ jobs:
       - name: Get operator logs
         run: kubectl logs --tail 100 -ntesting -loperator.juju.is/name=mlflow-server
         if: failure()
+
+  delete-unattached-volumes:
+    if: always()
+    uses: ./.github/workflows/delete-aws-volumes.yaml
+    secrets: inherit
+    with:
+      region: eu-central-1
```

Comment:
Could we define this as an ENV var that gets shared across the whole YAML? I see that other steps are using this as well

Reply:
I have nothing against the env, but would you mind if I dont do that  just to stay consistent with the kubeflow-bundle ci ?

Reply:
@misohu that means that we keep reproducing code we know we don't want, just to stay consistent with code we don't want :)

So no, let's fix this